.travis.yml

```diff
@@ -5,12 +5,17 @@ services:
 
 script:
     - cp docker-compose.yml.dist docker-compose.yml
+    - sed -i "s/REPLACE_WITH_YOUR_EMAIL/jeyser-crm.travis@example.com/g" docker-compose.yml
     - docker-compose build
+    # check prod env is working
     - docker-compose up -d
-#    - docker-compose run --rm web composer install -o -n
-    - sleep 10 && docker-compose run --rm web php bin/console cache:clear -e=test --no-warmup
+    - sleep 10 && docker-compose run --rm web php bin/console cache:clear -e=prod --no-warmup
```

Comment:
is docker necessary here? You have a VM you can freely configure, docker is a major slowdown

Reply:
Yes, it is.
We rely on Docker to provide a [5 command line installation](http://jeyser-crm.n7consulting.fr/dev/install#docker). It brought several Junior-Entreprise on board thanks to that feature. Travis checks that the install process is working for every commit.